sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/models/AmqpAddress.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.models;
+
+import java.util.Objects;
+
+/**
+ * This represents amqp address information. This will be used in populating information like 'To', 'ReplyTo' etc.
+ *
+ * <p><strong>Create and retrieve address</strong></p>
+ * {@codesnippet com.azure.core.amqp.models.AmqpAddress.createAndGet}
+ *
+ * @see <a href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-address-string">
+ *     Address type Format.</a>
```

Comment:
Have you checked that this renders correctly in generated JavaDoc?

Reply:
Yes, I did verify them.